server/database/queries/provider/getProviderById.js

```diff
@@ -0,0 +1,11 @@
+const connection = require('../../config/connection');
+
+const getProviderDataById = ({ id }) => {
+  const sql = {
+    text: 'SELECT * FROM providers WHERE id = $1;',
```

Comment:
there have a difference between id and user_id on the provider table, here you need the logged user (user_id), so you need to user_id, make sure that you get the right user